apycula/clock_fuzzer.py

```diff
@@ -266,9 +266,23 @@ def spine_aliases(quads, dests, clks):
                     aliases[spine_row, tap, dest] = ct[0], ct[1], dest
     return aliases
 
+# add border cells
+def add_rim(rows, cols):
+    if 1 in rows:
+        rows.add(0)
+    else:
+        # XXX fill all rows
+        rows.update({row for row in range(max(rows) + 1, db.rows)})
```

Comment:
What is going on here? Don't have the datastructure fresh in my mind

Reply:
When you define row and column ranges for quadrants, the rows that do not contain any DFF do not get into the list of quadrant rows, they are rows with memory apparently. Here I still add these rows so that then they have aliases.

Reply:
option is not very good because it adds only the outermost lines, but I do not have a good idea where the memory is usually located, I assume that at the edge of the quadrant.